dist/mapdc.js

```diff
@@ -48641,6 +48646,31 @@ function rasterChart(parent, useMap, chartGroup, _mapboxgl) {
     return _legend;
   };
 
+  _chart.deleteLayerLegend = function (currentLayer, deleteLayer, prevLayerId) {
```

Comment:
I'm guessing that `currentLayer` and `deleteLayer` are IDs like `prevLayerId`. If thats the case, maybe they should be named `currentLayerID` and such.